### PR TITLE
Fixed: Some grammar with the interactive doc.

### PR DESCRIPTION
**Pull request by artdude543**
Side note; this documentation is great. Well explained and the images help to tie into what the spec means. For quick recalling and referencing. Amazing work as always @ProbablePrime! 😃

### CODE REVIEW

File: src/reference/interactive/index.pug [hunk 1]
```diff
@@ -21,7 +21,7 @@ block reference
 
     h2#introduction Introduction
 
-    p Beam Interactive is a new way to let viewers directly control the environment in and around streamer’s broadcasts by letting viewers interact using a user interface. When a broadcast has interactivity enabled visual controls will appear beneath the broadcast's video that respond directly to the events of the broadcast. These controls can dynamically update as the broadcast progresses in response to different situations that occur.
+    p Beam Interactive is a new way to let viewers directly control the environment in and around streamer’s broadcasts by letting viewers interact using a user interface. When a broadcast has interactivity enabled visual controls will appear beneath the broadcast's video that responds directly to the events of the broadcast. These controls can dynamically update as the broadcast progresses in response to different situations that occur.
```
Comment (ProbablePrime) [inline src/reference/interactive/index.pug]: Not entirely sure about this one.

Why is responds better?

File: src/reference/interactive/index.pug [hunk 2]
```diff
@@ -297,7 +297,7 @@ block reference
         img(src='/img/reference/interactive/studio/share/shareCode.png' alt='studio share code')
     h4#explicit-sharing Explicit Sharing
 
-    p The third option, “Only allow specific users to play until published”, is explicit sharing it lets you share the project with named Beam users. Only users in the list can use the project. To share a project with a user search for their username in the search box and then click add.
+    p The third option, “Only allow specific users to play until published”. Means you can explicitly share the project with named Beam users. Only users in the list can use the project. To share a project with a user search for their username in the search box and then click add.
```
Comment (ProbablePrime) [inline src/reference/interactive/index.pug]: Yes!

File: src/reference/interactive/index.pug [hunk 3]
```diff
@@ -249,7 +249,7 @@ block reference
     p To add a control to the grid, click and drag the control from the Controls section on to the Grid section. Once the control has been added to the grid you have the ability to resize, reposition, or remove the control.
 
     p For more detail on the configuration of the controls on the scene, the #[code View JSON] option in the top right of the grid section will display the relevant JSON for the scene.
-    p Once you are finished building your scenes/controls, press the Save button in the top right of the grid section.
+    p Once you are finished building your scenes/controls press the Save button in the top right of the grid section.
```
Comment (ProbablePrime) [inline src/reference/interactive/index.pug]: I think the comma helps here

**Comment by ProbablePrime** [comment]
Thanks for the great feedback, Everyone on the team helped with these docs. Not just me :D give them all 🍰

File: src/reference/interactive/index.pug (hunk 2 above)
Comment (EthanWaite) [inline src/reference/interactive/index.pug]: > The third option, “Only allow specific users to play until published”. Means you can explicitly share the project with named Beam users.

One super small thing - shouldn't this be `...to play until published", means you can explicitly share...`

File: src/reference/interactive/index.pug (hunk 1 above)
Comment (connor4312) [inline src/reference/interactive/index.pug]: The previous version was correct. `The people respond to` vs `The people responds to`.

File: src/reference/interactive/index.pug [hunk 4]
```diff
@@ -237,7 +237,7 @@ block reference
 
     h4#controls-area Controls Area
 
-    p This is where you will manage the controls for the scenes in your project. Controls can be created, renamed, have their type (button or joystick) changed, spark cost adjusted, or deleted.
+    p This is where you will manage the controls for the scenes in your project. Controls can be created, renamed, have their type (button or joystick) changed, spark cost adjusted or deleted.
```
Comment (connor4312) [inline src/reference/interactive/index.pug]: Oxford commas plz 😛

File: src/reference/interactive/index.pug (hunk 3 above)
Comment (connor4312) [inline src/reference/interactive/index.pug]: We could use quotes around "Save", though...

File: src/reference/interactive/index.pug (hunk 1 above)
Comment (ProbablePrime) [inline src/reference/interactive/index.pug]: Ok Art, let's revert this change.

File: src/reference/interactive/index.pug (hunk 3 above)
Comment (ProbablePrime) [inline src/reference/interactive/index.pug]: Agreed can you add quotes Art?

**Comment by SimonSchick** [comment]
@artdude543 can you resolve the remaining requests?

**Comment by SimonSchick** [comment]
Closing due to inactivity.
